src/store/DiscussionStore.js

```diff
@@ -0,0 +1,64 @@
+import { defineStore } from "pinia"
```

Comment:
I wonder if the various stores could be refactored to share more code?
Not a priority, maybe I'll give it a shot later if you don't get to it.

Reply:
Probably! I'll see if I can refactor that later (if you didn't do it first ;-))